src/test/java/BaseTest.java

```diff
@@ -1,57 +1,18 @@
 import io.github.bonigarcia.wdm.WebDriverManager;
-import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.chrome.ChromeDriver;
 import org.openqa.selenium.firefox.FirefoxDriver;
 import org.openqa.selenium.safari.SafariDriver;
-import org.testng.annotations.AfterMethod;
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.BeforeSuite;
-import org.testng.annotations.Test;
 
-import java.time.Duration;
 
-// removed @Test annotation.
 public class BaseTest {
-    public static WebDriver driver = null;
 
     @BeforeSuite
-    static void setupClass() { WebDriverManager.chromedriver().setup(); }
-
-    @BeforeMethod
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/225115666-f3384664-3cb0-4fc3-888b-f12d3c7f220a.png)

**Hint**

**Step 1**  Please, remove code redundancies

Don't repeat yourself! ( In Java, it means don't write the same code repeatedly. )

The @BeforeMethod annotated method will be invoked before the execution of each test method where the test method is nothing but a test case.

![image](https://user-images.githubusercontent.com/104761482/225116624-40725e7a-e857-4096-bf5b-efc9729dfe9f.png)

**Step 2**

 Load your required URL using get()
 
 
![image](https://user-images.githubusercontent.com/104761482/225117619-f94d65bd-33cc-479c-b0bb-4dc41220cc73.png)

**Step 3**   After the Chrome 111 update, you can no longer kick off a chromedriver instance unless you add an additional chrome option:
"--remote-allow-origins=*"

```
        ChromeOptions options = new ChromeOptions();
        options.addArguments("--disable-notifications","--remote-allow-origins=*", "--incognito","--start-maximized");
        driver = new ChromeDriver(options);
```
![image](https://user-images.githubusercontent.com/104761482/225118015-fb81992f-f444-4c0a-836a-7f6f77d39da3.png)

Below are the list of available and most commonly used arguments for ChromeOptions class

**start-maximized:** Opens Chrome in maximize mode
**incognito:** Opens Chrome in incognito mode
**headless:** Opens Chrome in headless mode
**disable-extensions:** Disables existing extensions on Chrome browser
**disable-popup-blocking:** Disables pop-ups displayed on Chrome browser
**make-default-browser:** Makes Chrome default browser
**version:** Prints chrome browser version
**disable-infobars:** Prevents Chrome from displaying the notification ‘Chrome is being controlled by automated software

**SOLUTION:**

![image](https://user-images.githubusercontent.com/104761482/225119310-422752f8-590c-4446-b463-043ee18ff741.png)

![image](https://user-images.githubusercontent.com/104761482/225119395-f96e078c-a86f-43f8-b721-1e5737879808.png)